scripts/babel/default-options.js

```diff
@@ -10,8 +10,19 @@
 'use strict';
 
 var babelPluginModules = require('./rewrite-modules');
+var babelPluginAutoImporter = require('./auto-importer');
 var inlineRequires = require('./inline-requires');
-var plugins = [babelPluginModules];
+
+var plugins = [
+  {
+    position: 'after',
+    transformer: babelPluginAutoImporter,
+  },
+  {
+    position: 'after',
+    transformer: babelPluginModules,
+  },
+];
```

Comment:
Can we move this change into a second PR. That way we can ship the rest of this in a patch version